README.md

```diff
@@ -7,13 +7,13 @@ The current goals of j2me.js are:
 1. Run MIDlets in a way that emulates the reference implementation of phone ME Feature MR4 (b01)
 1. Keep j2me.js simple and small: Leverage the phoneME JDK/infrastructure and existing Java code as much as we can, and implement as little as possible in JavaScript
 
-## Install, Cold, and Warm runs
+## Install vs Hot runs
 
-J2ME.js launches MIDlets under three different circumstances. We are working to make the most common scenario the fastest, sometimes at the expense of making the least frequent scenarios a little slower.
+J2ME.js launches MIDlets under two different circumstances.
```

Comment:
We'll need to add a description for the *sharing content* and *notification* cases, but we can do that in a followup (once we actually implement them :smile:).
We'll also need to build a new benchmarking mode (probably very similar to the old warm benchmark) and a profiling mode for these new kinds of startup (which is also useful to accurately separate the startup time between BG startup time and FG startup time).